cypress/integration/courses.spec.js

```diff
@@ -0,0 +1,74 @@
+describe('courses page', () => {
+  before(() => {
+    cy.loginAndVisit();
+  });
+
+  it('navigates to Courses page from Summary', () => {
+    cy.contains('Courses').click();
+    cy.location().should(loc => {
+      expect(loc.pathname).to.contain('courses');
+    });
+  });
+
+  it('contains a courses table', () => {
+    cy.get('table[course-table]');
+  });
+
+  it('removes ECS courses from 2018-19', () => {
+    cy.get('[data-event-type=selectAllCourseRows].checkbox-container').click();
+    cy.get('span.tool-icon.glyphicon.glyphicon-trash').click();
+    cy.get('.delete-course-modal-footer').within($modal => {
+      cy.server();
+      cy.route('PUT', '**/courseView/**').as('deleteCourses');
+      cy.get('button.btn-danger').click();
+    });
+    cy.wait('@deleteCourses');
+  });
+
+  it('adds ECS courses from 2015-16', () => {
+    cy.get('.glyphicon-plus').click();
```

Comment:
Do we need a more specific selector for this? .glyphicon-plus could easily end up elsewhere on a page.